dev-requirements.txt

```diff
@@ -1,50 +1,139 @@
 -i https://pypi.python.org/simple
-apipkg==1.5
+apipkg==1.5; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
 appdirs==1.4.4
-attrs==20.1.0
+asn1crypto==1.4.0
+attrs==20.1.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+autobahn==20.7.1; python_version >= '3.5'
+automat==20.2.0
 bandit==1.6.2
+base58==2.0.1; python_version >= '3.5'
+bitarray==1.2.2
+blake2b-py==0.1.3
+bytestring-splitter==2.2.0
+cached-property==1.5.1
 certifi==2020.6.20
-cfgv==3.2.0
+cffi==1.14.2
+cfgv==3.2.0; python_full_version >= '3.6.1'
 chardet==3.0.4
+click==7.1.2
+coincurve==13.0.0
+colorama==0.4.3
+constant-sorrow==0.1.0a9
+constantly==15.1.0
 coverage==5.2.1
+cryptography==3.1
+cytoolz==0.10.1; implementation_name == 'cpython'
+dateparser==0.7.6; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
```

Comment:
it's strange that this extra `python_version` text is being added ...

Reply:
Yes, I agree - was this build from OSX, or docker?

Reply:
I give up.
I did exactly what @KPrasch said to do on discord.
in docker:
```
root@5a974c44fed9:/code# rm Pipfile.lock 
root@5a974c44fed9:/code# rm requirements.txt 
root@5a974c44fed9:/code# rm dev-requirements.txt 
root@5a974c44fed9:/code# make lock
```

Reply:
Ill give it a try - I've been having good success making builds.

Reply:
Actually everything is working now... maybe this is just some new pip thing.

Reply:
I just rebuilt the docker image so it's fresh, now I'm trying it again and seeing what happens